### ISSUE TITLE
[1.x] Enable to set default codecs for videorooms

### ISSUE THREAD

**Issue by Gymnae** [meetecho/janus-gateway#3205]
**What version of Janus is this happening on?**
v1.1.3

**Have you tested a more recent version of Janus too?**
no
**Was this working before?**
new request

**Additional context**
I'm using janus for the https://github.com/strukturag/nextcloud-spreed-signaling backend. It creates new rooms on the fly and I have not hand in editing the command which is used to create rooms.

Thus I'd like to be able to define default audio/video codecs which are then added to all newly created rooms. Right now this is a per room setting, but since I'm using janus only with the nextcloud talk signaling backend, I can't adjust them.

**Comment by lminiero** [comment]
Sounds like a request you may want to ask the Nextcloud developers, not us. In Janus you can already configure that.

**Comment by Gymnae** [comment]
You are correct, I created a PR in their repo to by default create rooms with VP,h265,VP8,h264 (in that order)
